.spelling

```diff
@@ -820,3 +820,6 @@ v6.0.
  - docs/host-powershell/README.md
 CorePsAssemblyLoadContext.cs
 sample-dotnet1
+NTLM-based
+Kerberos-based
+post-6
```

Comment:
It's not showing up because of the force push I did, but I assure you it's added.